subiquity/common/apidef.py

```diff
@@ -412,8 +414,17 @@ def GET() -> Optional[ADConnectionInfo]: ...
         # simultaneously - such as invalid chars on the admin name and DC name
         # starting with a hyphen or a dot. Thus this must returns a List
         # of errors [ADValidationResult.OK] on success.
-        def POST(data: Payload[ADConnectionInfo]) \
-            -> List[ADValidationResult]: ...
+        def POST(data: Payload[ADConnectionInfo]) -> None: ...
```

Comment:
I think this comment is a bit out of date now?

Reply:
Indeed :D Thanks for catching that!